README.md

```diff
@@ -105,15 +105,122 @@ always in sync, and that one or the other may be slightly more up-to or out-of d
+Not ontologized. These types are ignored due to the uncertainty of the nature of the association.
+
+2: The disorder has been placed on the map by linkage or other statistical method; no mutation has been found.  
+[RO:0003303 (causes condition)](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003303): 
+A relationship between an entity (e.g. a genotype, genetic variation, chemical, or environmental exposure) and a 
+condition (a phenotype or disease), where the entity has some causal role for the condition.
+
+3: The molecular basis for the disorder is known; a mutation has been found in the gene.  
+[RO:0004013 (is causal germline mutation in)](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0004013): 
+Relates a gene to condition, such that a mutation in this gene is sufficient to produce the condition and that can be 
+passed on to offspring[modified from orphanet].
+
+Note: For these "mapping key (3)" cases, there also exists an inverse predicate which we ontologize in the 
+inverse direction: (Gene MIM) --(Mapping key 3 / RO:0004003)--> (Disease MIM):
+[RO:0004003 (has material basis in germline mutation in)](https://www.ebi.ac.uk/ols4/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0004003) 
+
+4: A contiguous gene deletion or duplication syndrome, multiple genes are deleted or duplicated causing the phenotype.  
+[RO:0003304 (contributes to condition)](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003304): 
+A relationship between an entity (e.g. a genotype, genetic variation, chemical, or environmental exposure) and a 
+condition (a phenotype or disease), where the entity has some contributing role that influences the condition.
+
+**Important caveat: Singular vs multiple associations**
+These above RO predicates are only used if there is only 1 gene associated with a given disease, i.e. 
+in `morbidmap.txt`, there is only 1 row where the MIM appears in the `Phenotype` field.
+
+In cases where there is >1 association, the following RO predicate is used instead, regardless of if the mapping key is 
+(2), (3), or (4):
+[RO:0003302 (causes or contributes to condition)](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0003302): 
+A relationship between an entity (e.g. a genotype, genetic variation, chemical, or environmental exposure) and a 
+condition (a phenotype or disease), where the entity has some causal or contributing role that influences the condition.
+
+#### Necessary conditions for disease-defining associations (RO:0004013)
+Of the above 3 Disease->Gene association predicates (those with mapping keys (2), (3), and (4)), the one which we 
+consider "disease defining" is (3) (RO:0004013). For this association, we also have other conditions that we check. If 
```

Comment:
I thought Disease->Gene used the conceptually inverse property of RO:0004003 or am I reading this backwards?

Reply:
You are right. I have it flipped, as well as an above section as well. I need to fix that.